sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md

```diff
@@ -2,5 +2,75 @@
 
 ## 1.0.0-preview.1
 
-### New Features
-- Initial preview of Azure Management KeyVault SDK based on Azure.Core
\ No newline at end of file
+This is public preview version based on Azure Core. This version uses a next-generation code generator that introduces important new features.
```

Comment:
This package follows the new Azure SDK guidelines which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.

This is a Public Preview version, so expect incompatible changes in subsequent releases as we improve the product.  To provide feedback, please submit an issue in our Azure SDK for .NET GitHub repo.

Reply:
Will update with sdk repo url